src/components/HandleErrorsComponent.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from 'react';
+import { AddingError } from './Notifications/AddingError';
+import { DeleteError } from './Notifications/DeleteError';
+import { ErrorLoadingComments } from './Notifications/ErrorLoadingComments';
+import { NoCommentsYet } from './Notifications/NoCommentsYet';
+
+type Props = {
+  isLoadingError: boolean,
+  isDeleteError: boolean,
+  isAddingError: boolean,
+  isNoComments: boolean,
+  isLoader: boolean,
+};
+
+export const HandleErrorsComponent: FC<Props> = ({
+  isLoadingError,
+  isDeleteError,
+  isAddingError,
+  isNoComments,
+  isLoader,
+}) => {
+  return (
+    <>
+      {isLoadingError && (
+        <ErrorLoadingComments />
+      )}
+
+      {isDeleteError && (
+        <DeleteError />
+      )}
+
+      {isAddingError && (
+        <AddingError />
+      )}
+
+      {isNoComments && !isLoader && (
+        <NoCommentsYet />
+      )}
```

Comment:
Are you sure you need separate components for each error message? Maybe would be better to create some array with the necessary values? And use a `map` for rendering it?